cg-spec/editors_draft.html

```diff
@@ -941,7 +941,206 @@ <h2>SPARQL-star Query Results XML Format</h2>
+
+        <p>Notice that deleting triples by using the DELETE DATA operation does not affect any other triples in the corresponding graphs. For instance, for a default graph that contains the triple <code>:bob :age 23</code> as an asserted triple, the DELETE DATA operation given above does <em>not</em> delete this asserted triple. In contrast, the following operation would delete this asserted triple but it would <em>not</em> delete any nested triple that contains the given triple as an embedded triple.</p>
+
+        <pre data-transform="updateExample"
+             data-content-type="application/x-sparql-star-query"
+             class="nohighlight example"
+        >
+          <!--
+          PREFIX : <http://www.example.org/>
+
+          DELETE DATA {
+            :bob :age 23 .
+          }
+          -->
+        </pre>
+
+      </section>
+
+      <section>
+        <h2>DELETE/INSERT</h2>
+
+        <p>The <a data-cite="SPARQL11-UPDATE#deleteInsert">DELETE/INSERT</a> operation can be used to remove or add triples based on variable bindings obtained by evaluating a given WHERE clause. As an example, the following DELETE/INSERT operation replaces all nested triples in which <code>:alice</code> is the subject by nested triples in which <code>:alicia</code> is the subject.</p>
+
+        <pre data-transform="updateExample"
+             data-content-type="application/x-sparql-star-query"
+             class="nohighlight example"
+        >
+          <!--
+          PREFIX : <http://www.example.org/>
+
+          DELETE { :alice  ?pp <<?s ?p ?o>> . }
+          INSERT { :alicia ?pp <<?s ?p ?o>> . }
+          WHERE {  :alice  ?pp <<?s ?p ?o>> . }
+          -->
+        </pre>
+
+        <p>As in SPARQL Update, in SPARQL-star Update it is possible to use variations of the DELETE/INSERT operations in which either the DELETE clause or the INSERT clause are omitted.</p>
+
+        <p>When removing triples via the DELETE clause (irrespective of whether the INSERT clause is omitted or not), the effects with respect to nested triples must be the same as described above for the DELETE DATA operation; i.e., only the triples that are explicitly identified to be deleted are deleted. Similarly, inserting triples via the INSERT clause has the same effects as described above for the INSERT DATA operation; i.e., inserting nested triples does not automatically also insert their embedded triples as asserted triples into the graph. Of course, it is possible to request such inserts explicitly as demonstrated in the following example.</p>
+
+        <pre data-transform="updateExample"
+             data-content-type="application/x-sparql-star-query"
+             class="nohighlight example"
+        >
+          <!--
+          PREFIX : <http://www.example.org/>
+
+          DELETE { :alice  ?pp <<?s ?p ?o>> . }
+          INSERT { :alicia ?pp <<?s ?p ?o>> .  ?s ?p ?o .}
+          WHERE {  :alice  ?pp <<?s ?p ?o>> . }
+          -->
+        </pre>
```

Comment:
(for later?) Include the annotation version of this example.

Reply:
Yes, let's keep this for later (once the issue about the annotation syntax for the query language is settled)